consensus/src/sync/history/sync.rs

```diff
@@ -8,6 +8,7 @@ use futures::{future::BoxFuture, stream::FuturesUnordered, FutureExt};
 use nimiq_blockchain::Blockchain;
 use nimiq_hash::Blake2bHash;
 use nimiq_network_interface::network::{Network, SubscribeEvents};
+use nimiq_utils::WakerExt as _;
```

Comment:
```suggestion
use nimiq_utils::WakerExt;
```

I don't think the import has to be marked as `_`.

Reply:
Importing it as `_` is indeed not necessary.

By importing it as `_` (i.e. not giving this import any name, `_` is special), I make sure that the `WakerExt` symbol does not unnecessarily get imported into the module when it is actually unused. I just want to import the trait to be able to call its methods.